modules/ROOT/pages/dataweave-formats-ndjson.adoc

```diff
@@ -106,43 +109,66 @@ The JSON output is an array of the valid objects from the ndjson input.
 ]
 ----
 
+// CONFIG PROPS GET APPENDED BELOW /////////////////////////////////
 
-// CONFIG PROPS ///////////////////////////////////////////////////////
 
 [[properties]]
 == Configuration Properties
 
-DataWeave supports the following configuration properties for the Newline Delimited JSON format (ndjson).
+DataWeave supports the following configuration properties for this format.
 
+
+[[reader_properties]]
 === Reader Properties
 
-The ndjson format accepts properties that provide instructions for reading input data.
+This format accepts properties that provide instructions for reading input data.
+
 
 [cols="1,1,1,3a", options="header"]
 |===
-| Parameter | Type | Default | Description
-| `ignoreEmptyLine` | `Boolean` | `true` | Ignores any empty line. Valid values are `true` or `false`.
-| `skipInvalid` | `Boolean` | `false` | Skips data that is not valid ndjson. Valid values are `true` or `false`.
+|Parameter |Type |Default|Description
+|`ignoreEmptyLine`  |`Boolean`|`true`|Ignores any empty line.
+ Valid values are `true` or `false`.
+|`skipInvalid`  |`Boolean`|`false`|Skips invalid records and ignores values that are not valid JSON.
+ Valid values are `true` or `false`.
 |===
 
+[[writer_properties]]
 === Writer Properties
 
-The ndjson format accepts properties that provide instructions for writing output data.
+This format accepts properties that provide instructions for writing output data.
 
 [cols="1,1,1,3a", options="header"]
 |===
-| Parameter | Type | Default | Description
-| `bufferSize` | `Number` | `8192` | Size of the writer buffer.
-| `deferred` | `Boolean` | `false` | When set to `true`, DataWeave generates the output as a data stream, and the script's execution is deferred until it is consumed. Valid values are `true` or `false`.
-| `encoding` | `String` | `'UTF-8'` | Encoding for the ndjson reader to use.
-| `skipNullOn` | `String` | `null` | Valid values are `arrays` or `objects`.
-| `writeAttributes` | `Boolean` | `false` | Valid values are `true` or `false`.
+|Parameter |Type |Default|Description
+|`bufferSize`  |`Number`|`8192`|Size of the buffer writer.
+|`deferred`  |`Boolean`|`false`|Generates the output as a data stream when set to `true`, and defers the script's execution until consumed.
+ Valid values are `true` or `false`.
+|`encoding`  |`String`|`'UTF-8'`|The character set to use for the output, such as UTF-8.
+|`skipNullOn`  |`String`|`null`|Skips `null` values in the specified data structure. By default,
```

Comment:
TODO (see above?): skipNullOn copyedit "it does not" -> "DataWeave does not"? 

and fix formatting in HTML of "Valid values are..."